internal/proxy/validate_util.go

```diff
@@ -135,6 +140,127 @@ func (v *validateUtil) checkAligned(data []*schemapb.FieldData, schema *typeutil
 	return nil
 }
 
+func (v *validateUtil) fillWithDefaultValue(data []*schemapb.FieldData, schema *typeutil.SchemaHelper, numRows uint64) error {
+	for _, field := range data {
+		fieldSchema, err := schema.GetFieldFromName(field.GetFieldName())
+		if err != nil {
+			return err
+		}
+		// if default value is not set, continue
+		// compatible with 2.2.x
+		if fieldSchema.GetDefaultValue() == nil {
+			continue
```

Comment:
What will happen if the insert batch lacks some field and the default value is not set as this line doesn't return error?

Reply:
it will be completed in the sdk. For example, in pymilvus, in the process of data->entities, it is read in the order of the field schema. If the column is not passed or None is passed in, it is automatically considered that default should be used, and it is passed in according to the name and type corresponding to the schema here. If in the go sdk, a certain column is not passed in, the same is true. It is considered that default should be used here, and parameters such as name are taken from the field schema.